nessus-agent/windows/init.sls

```diff
@@ -6,45 +6,21 @@
 # In addiont, the sls file requires that the salt winrepo has been
 # configured with a `nessus agent` pkg definition
 
-{%- if nessus.log_rotation_strategy %}
-Add Log Rotation Strategy:
+{%- for log_file,log_rotation in nessus.log_config.items() %}
+  {%- if log_rotation %}
```

Comment:
set the default log_config to an empty dict, and this if is no longer needed... `log_config: {}`

Reply:
Good to know.  Didn't realize that.